pathspec/tests/test_util.py

```diff
@@ -367,3 +368,24 @@ def test_3_entries(self):
 			'Dir/Inner/f',
 			'Empty',
 		])))
+
+	@unittest.skipIf(sys.version_info < (3, 4), "pathlib entered stdlib in Python 3.4")
+	def test_4_normalizing_pathlib_path(self):
+		"""
+		Tests passing pathlib.Path as argument.
+		"""
+		from pathlib import Path
+		first_spec = normalize_file(Path('a.txt'))
+		second_spec = normalize_file('a.txt')
+		self.assertEqual(first_spec, second_spec)
+
+	@unittest.skipIf(sys.version_info < (3, 4), "pathlib entered stdlib in Python 3.4")
+	def test_4_normalizing_pathlib_paths(self):
```

Comment:
Due to my comment on *util.normalize_files()* this test won't be needed.

Reply:
Done via 77521fe